packages/p2p/src/components/advertiser-page/advertiser-page-adverts.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import classNames from 'classnames';
+import { InfiniteDataList, Loading, Table, Tabs } from '@deriv/components';
+import { isDesktop, isMobile, useIsMounted } from '@deriv/shared';
+import { observer } from 'mobx-react-lite';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import Empty from 'Components/empty/empty.jsx';
+import AdvertiserPageRow from './advertiser-page-row.jsx';
+import './advertiser-page.scss';
+
+const AdvertiserPageAdverts = () => {
+    const { advertiser_page_store, general_store } = useStores();
+    const isMounted = useIsMounted();
+    const previous_scroll_top = React.useRef(0);
+
+    const onScroll = event => {
+        if (isMounted() && event.target.scrollTop !== previous_scroll_top.current) {
+            const is_scrolling_down = event.target.scrollTop > previous_scroll_top.current;
+            if (is_scrolling_down) {
+                if (advertiser_page_store.is_scroll_effect_enabled) {
+                    advertiser_page_store.setIsStatsVisible(false);
+                }
+            } else {
+                // if (!advertiser_page_store.is_scroll_effect_enabled) {
+                //     advertiser_page_store.setIsStatsVisible(true);
+                //     advertiser_page_store.setIsScrollEffectEnabled(true);
+                // }
```

Comment:
This is intentional?

Reply:
Definitely not, remnants from trying to make multiple scrolls work. Removing it :)